corehq/apps/userreports/tasks.py

```diff
@@ -46,7 +46,9 @@ def _get_db(doc_type):
 
 
 # This is intentionally not using magic to introspect the class from the name, though it could
+from corehq.apps.locations.models import Location
 _DOC_TYPE_MAPPING = {
     'XFormInstance': XFormInstance,
     'CommCareCase': CommCareCase,
+    'Location': Location
```

Comment:
I'm not worried about porting to SQL when ready/necessary. As long as we can recreate a json structure for a location then 99% of the code can stay the same. Agree it would be great to get rid of couch locations, and I'd be pro you or someone leading that (and happy to deal with the UCR side of things when that's ready)